pkg/agent/openflow/client.go

```diff
@@ -406,7 +406,7 @@ func (c *client) InstallNodeFlows(hostname string,
 		if peerGatewayIP.To4() != nil {
 			// Since broadcast is not supported in IPv6, ARP should happen only with IPv4 address, and ARP responder flows
 			// only work for IPv4 addresses.
-			flows = append(flows, c.arpResponderFlow(peerGatewayIP, cookie.Node))
+			flows = append(flows, c.arpResponderFlow(peerGatewayIP, globalVirtualMAC, priorityNormal, cookie.Node))
```

Comment:
Updated.

Reply:
Updated.

Reply:
Reminder.

Reply:
Sorry I didn't reply to this comment but I updated it. It is not related to Service traffic. The change was just a result of refactoring `arpResponderFlow()`

However, now, to keep it same as Linux, new arp responder flows are unnecessary because we can use `NetNeighbor` on Windows to deal with neighbor cache. So I changed it back now but keep the comment, which looks like:
"arpResponderFlow() uses globalVirtualMAC for gateway IP as a virtual destination MAC address. This ARP responder is for request traffic from local going through tunnel to different Nodes."